### ISSUE TITLE
validation on param converter

### ISSUE THREAD

**Issue by sergio11** [AndrewKeig/express-validation#34]
Hi,

How I can use validation in a param converter?
I wanted to use Joi.objectId () to validate that this is a mongodb valid id 

```
router.route('/:id')
  /** GET /api/users/:userId - Get user */
  .get(validate(paramValidation.get),userCtrl.get)

  /** PUT /api/users/:userId - Update user */
  .put(validate(paramValidation.updateUser), userCtrl.update)

  /** DELETE /api/users/:userId - Delete user */
  .delete(validate(paramValidation.deleteUser), userCtrl.remove);

/** Load user when API with userId route parameter is hit */
router.param('id', userCtrl.load);
```

Thanks and regards

**Comment by colthreepv** [comment]
I think you can just use:

`router.param('id', validate(paramValidation. user), userCtrl.load);`

The param callbacks receive the same middleware structure as full-fledged
routes

Il 06/ago/2016 13:42, "Sergio Sánchez Sánchez" <notifications@github.com>
ha scritto:

**Comment by sergio11** [comment]
Hi,

If I do that , the converter param callback (userCtrl.load) is not called.

thanks for answering

**Comment by AndrewKeig** [comment]
Closing, released 3.0
